go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -1280,10 +1290,11 @@ func (be *BuiltinBackupEngine) restoreFile(ctx context.Context, params RestorePa
 	params.Stats.Scope(stats.Operation("Destination:Open")).TimedIncrement(time.Since(openDestAt))
 
 	defer func() {
-		closeDestAt := time.Now()
-		if cerr := dest.Close(); cerr != nil {
+		if cerr := closeWithRetry(ctx, params.Logger, dest, fe.Name); cerr != nil {
 			finalErr = errors.Join(finalErr, vterrors.Wrap(cerr, "failed to close destination file"))
+			params.Logger.Errorf("Failed to close destination file %s during restore: %v", dest.Name(), err)
 		}
+		closeDestAt := time.Now()
```

Comment:
I think a refactor means this isn't measuring the `Close()` now, because it's after `closeWithRetry`

Reply:
I cleaned this up here: https://github.com/vitessio/vitess/pull/18848/commits/b2f776bbe28af1ba02cd9082a82d373081eae263